### PR TITLE
Added : customize the color and background color of selected text

### PR DESCRIPTION
**Pull request by rizwan3395**

### CODE REVIEW

File: frontend/.vscode/settings.json
Comment (LucasXu0) [inline frontend/.vscode/settings.json:25]: unnecessary change. Please reset it.

File: frontend/app_flowy/assets/translations/en.json
Comment (LucasXu0) [inline frontend/app_flowy/assets/translations/en.json:140]: Same as previous.

File: frontend/app_flowy/lib/plugins/board/presentation/board_page.dart
Comment (LucasXu0) [inline frontend/app_flowy/lib/plugins/board/presentation/board_page.dart:23]: Same as previous.

File: frontend/app_flowy/packages/appflowy_editor/lib/src/extensions/text_node_extensions.dart
Comment (LucasXu0) [inline frontend/app_flowy/packages/appflowy_editor/lib/src/extensions/text_node_extensions.dart:36]: Please add an empty line before 35.

File: frontend/app_flowy/packages/appflowy_editor/lib/src/extensions/text_node_extensions.dart
Comment (LucasXu0) [inline frontend/app_flowy/packages/appflowy_editor/lib/src/extensions/text_node_extensions.dart:41]: Please add an empty line before 40.

File: frontend/app_flowy/packages/appflowy_editor/lib/src/render/color_menu/color_menu.dart
Comment (LucasXu0) [inline frontend/app_flowy/packages/appflowy_editor/lib/src/render/color_menu/color_menu.dart:11]: Can't import the additional packages. Because appflowy_editor doesn't depend on them.

1. package:flowy_sdk
2. package:app_flowy

File: frontend/app_flowy/packages/appflowy_editor/lib/src/render/emoji/emoji_select_widget.dart
Comment (LucasXu0) [inline frontend/app_flowy/packages/appflowy_editor/lib/src/render/emoji/emoji_select_widget.dart:1]: It seems that you didn't merge the origin/main before coding. This file has been moved to package `appflowy_editor_plugin `. So please don't submit it again.

File: frontend/app_flowy/packages/flowy_infra_ui/flowy_infra_ui_platform_interface/lib/src/method_channel_flowy_infra_ui.dart
Comment (LucasXu0) [inline frontend/app_flowy/packages/flowy_infra_ui/flowy_infra_ui_platform_interface/lib/src/method_channel_flowy_infra_ui.dart:4]: Don't modify the code with the package `flowy_infra_ui`.

**Review by LucasXu0 — CHANGES_REQUESTED** [review]

**Comment by LucasXu0** [comment]
Hi, @rizwan3395. After some simple tests, I find that the color menu can't be scrolled. That need to be fixed before merging.

File: frontend/app_flowy/packages/appflowy_editor/lib/src/render/color_menu/color_menu.dart
Comment (rizwan3395) [reply frontend/app_flowy/packages/appflowy_editor/lib/src/render/color_menu/color_menu.dart:11]: These packages are for colours list. Should I create the new list of with name and then use that ?

**Pull request merged**
